assets/app/features/upgrade/controllers/upgrade-upgrade-administration-server.controller.spec.js

```diff
@@ -65,6 +65,44 @@ describe('Upgrade Flow - Upgrade Administration Server Controller', function ()
         should.exist(controller);
     });
 
+    describe('on controller creation', function () {
+        afterEach(function () {
```

Comment:
It could be reset in the top level `beforeEach()` and it should be clean for every test no matter where it's executed. See http://coderdiaries.com/2014/05/30/jasminejs-execution-sequence-whith-nested-describes/